src/autoscaler/api/cmd/api/main.go

```diff
@@ -61,15 +61,15 @@ func main() {
 	}
 	defer policyDb.Close()
 
-	if !conf.UseBuildInMode {
-		var bindingDB db.BindingDB
-		bindingDB, err = sqldb.NewBindingSQLDB(conf.DB.BindingDB, logger.Session("bindingdb-db"))
-		if err != nil {
-			logger.Error("failed to connect bindingdb database", err, lager.Data{"dbConfig": conf.DB.BindingDB})
-			os.Exit(1)
-		}
-		defer bindingDB.Close()
+	var bindingDB db.BindingDB
+	bindingDB, err = sqldb.NewBindingSQLDB(conf.DB.BindingDB, logger.Session("bindingdb-db"))
+	if err != nil {
+		logger.Error("failed to connect bindingdb database", err, lager.Data{"dbConfig": conf.DB.BindingDB})
+		os.Exit(1)
+	}
+	defer bindingDB.Close()
```

Comment:
when it is built-in mode, it seems not necessary to initialize bindingDB. please double check.